openapi.yml

```diff
@@ -387,6 +391,22 @@ components:
           example: Searchworks
         release:
           type: boolean
+    MessageDigest:
+      description: The output of the message digest algorithm.
+      type: object
+      properties:
+        type:
+          description: The algorithm that was used
+          type: string
+          enum:
+            - md5
+            - sha1
+        digest:
```

Comment:
Can we call this `value` instead to conform to premis (circa-2017) which uses `rdf:value`?